pkg/rancher/migrations_test.go

```diff
@@ -183,8 +184,11 @@ func TestVersionTombstone(t *testing.T) {
 				return obj, nil
 			}).AnyTimes()
 
-			tt.setup(cm, configMapController, configMapCache)
-			err := versionTombstone(configMapController)
+			tt.setup(t, cm, configMapController, configMapCache)
+			if tt.currentVersion != "" {
+				rancherversion.Version = tt.currentVersion
```

Comment:
Do we still need to set this if the tombstone function now accepts the version as a parameter?

Reply:
yes, because of this: https://github.com/rancher/rancher/blob/main/pkg/rancher/migrations.go#L142-L144. I still left the explicit `"dev"` version check inside of `versionTombstone` even though it's unreachable because I wanted to be explicit that it really should never attempt to validate in a dev environment. I can remove if you think it's unnecessary.